### DISCUSSION TITLE
Where Can I Find Gestational Age Data for Neonates in MIMIC-IV Version 1.0?

### DISCUSSION THREAD

**Discussion by karinfridkin** [MIT-LCP/mimic-code#1775] [MIMIC-General]
Hi everyone,

I’m working with MIMIC-IV version 1.0 for research on neonatal babies and need to find their gestational age. Could someone guide me on where to locate this information in the dataset? I’ve come across a paper that utilized this data, so it should be included somewhere.

Thank you!

**Comment by alistairewj** [comment]
MIMIC-IV v1.0 doesn't include any of the ICU data for neonates, so you probably won't be able to do a study on them using that data.
